client/my-sites/stats/activity-log/threat-alert.jsx

```diff
@@ -143,25 +144,25 @@ export class ThreatAlert extends Component {
 										compact
 										primary
 										label={ translate( 'Fix threat' ) }
-										onClick={ () => console.log( 'main button clicked' ) }
+										onClick={ noop }
```

Comment:
It seems that this was left out there with debug purposes. I'd suggest using `debug` instead, so that we could keep the same purpose that this was created with.